### ISSUE TITLE
VehicleModelType as command parameter

### ISSUE THREAD

**Issue by LDami** [ikkentim/SampSharp#416]
With the following code, `/v tug` does not work (Usage message displayed), but `/v Tug` works.

```cs
[Command("vehicle", "veh", "v", DisplayName = "v")]
private static void SpawnVehicleCommand(Player player, VehicleModelType model)
{
 // Spawn vehicle
}
```

This might be caused by 2 entries in the enum starting with same name: Tug and TugStairsTrailer.

https://github.com/ikkentim/SampSharp/blob/master/src/SampSharp.GameMode/SAMP/Commands/ParameterTypes/EnumType.cs#L59

**Comment by VillainOP2393** [comment]
if(sscanf(params, "s[20]I(-1)I(-1)", model, color1, color2))
	{
	    return SCM(playerid, COLOR_SYNTAX, "Usage: /veh [modelid/name] [color1 (optional)] [color2 (optional)]");
	}

**Comment by CiprianN23** [comment]
> if(sscanf(params, "s[20]I(-1)I(-1)", model, color1, color2))
> 	{
> 	    return SCM(playerid, COLOR_SYNTAX, "Usage: /veh [modelid/name] [color1 (optional)] [color2 (optional)]");
> 	}

Hi

Your code snippet is from pawn, it is not relevant in the context of SampSharp and C# and also is not working in this context

**Comment by ikkentim** [comment]
I tested this with this command, but it works fine for me:
```cs
[Command("vehicle", "veh", "v", DisplayName = "v")]
private static void SpawnVehicleCommand(Player player, VehicleModelType model)
{
    // Spawn vehicle
    var vehicle = Vehicle.Create(model, player.Position, 0, 0, 0);
    player.PutInVehicle(vehicle);
}
```

Could you provide some repro case?

**Comment by LDami** [comment]
Fixed in 0.10
